src/main/java/porcelli/me/git/integration/githook/push/integration/BitBucketIntegration.java

```diff
@@ -0,0 +1,95 @@
+package porcelli.me.git.integration.githook.push.integration;
+
+import java.util.List;
+
+import org.eclipse.jgit.transport.CredentialsProvider;
+import org.eclipse.jgit.transport.UsernamePasswordCredentialsProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.aliction.gitproviders.bitbucket.client.BitbucketV2API;
+import com.aliction.gitproviders.bitbucket.exceptions.BitbucketCreateRepositoryException;
+import com.aliction.gitproviders.bitbucket.exceptions.BitbucketGetTeamException;
+import com.aliction.gitproviders.bitbucket.exceptions.BitbucketPageException;
+import com.aliction.gitproviders.bitbucket.objects.BitbucketRepository;
+import com.aliction.gitproviders.bitbucket.objects.BitbucketRole;
+import com.aliction.gitproviders.bitbucket.objects.BitbucketTeam;
+import com.aliction.gitproviders.bitbucket.objects.BitbucketUser;
+
+import porcelli.me.git.integration.githook.push.properties.GitRemoteProperties;
+
+public class BitBucketIntegration implements GitRemoteIntegration {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(BitBucketIntegration.class);
+
+    private static final String ERROR_LOGIN = "Connecting using username and password is not supported with Bitbucket Cloud, kindly use token instead.";
+
+    private BitbucketV2API bitbucket;
+    private CredentialsProvider credentialsProvider;
+    private BitbucketTeam team = null;
+    private BitbucketUser user;
+    private String providerURL;
```

Comment:
this property is not used, just delete it

Reply:
Yeah, this was used for building url, but I used links later and forget to remove.
Deleted